src/app/submission/submission.service.spec.ts

```diff
@@ -355,15 +359,24 @@ describe('SubmissionService test suite', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       imports: [
-        StoreModule.forRoot({ submissionReducers })
+        StoreModule.forRoot({ submissionReducers }),
+        TranslateModule.forRoot({
+          loader: {
+            provide: TranslateLoader,
+            useClass: MockTranslateLoader
+          }
+        })
       ],
       providers: [
         { provide: GLOBAL_CONFIG, useValue: config },
         { provide: Router, useValue: router },
         { provide: SubmissionRestService, useValue: restService },
         { provide: ActivatedRoute, useValue: new MockActivatedRoute() },
+        NotificationsService,
```

Comment:
Please try to mock any external services you use in tests. See https://github.com/DSpace/dspace-angular/issues/281
If we use actual services, the tests not only become harder to write, but they can break when the external service changes. They can also influence and break other, seemingly unrelated tests, causing issues that are very hard to debug. (e.g. tests that fail only when a specific other, unrelated test has run first)